expense/src/workflows.ts

```diff
@@ -0,0 +1,41 @@
+import { Expense, ExpenseStatus } from './interfaces';
+import { Trigger, sleep } from '@temporalio/workflow';
+import { createActivityHandle } from '@temporalio/workflow';
+import type * as activities from './activities';
+
+const { createExpense, payment } = createActivityHandle<typeof activities>({
+  scheduleToCloseTimeout: '5 minutes',
+});
+
+let status: ExpenseStatus = ExpenseStatus.CREATED;
```

Comment:
While this is totally fine and supported by the sdk I think we should discourage using mutable global state.
In the future we might want to run workflows in environments that are not isolated